plugins/slipping-away-users/frontend/public/templates/SlippingAwayUsersPage.html

```diff
@@ -0,0 +1,79 @@
+            <slipping-away-users-bar-chart> </slipping-away-users-bar-chart>
+            <slipping-away-users-table> </slipping-away-users-table>
+        </cly-main>
+    </div>
+</script>
+
+<!-- NOTE-LA: <slipping-away-users-filter> is a temporary created component until query filter with modal becomes available-->
+<script type="text/x-template" id="slipping-away-users-filter">
+    <cly-section style="min-height:150px">
+        <div class="bu-columns bu-is-gapless bu-is-mobile bu-is-centered" style="padding-top:1%">
+            <div class="bu-column bu-is-12-tablet bu-is-9-desktop bu-is-7-widescreen bu-is-6-fullhd">
+                <cly-qb-segmentation
+                    :allow-breakdown="false"
+                    v-model="slippingAwayUsersFilters">
+                </cly-qb-segmentation>
+            </div>
+        </div>
+        <div class="bu-columns bu-is-gapless bu-is-mobile bu-is-centered" style="padding-bottom:1%">
+            <div class="bu-column bu-is-12-tablet bu-is-9-desktop bu-is-7-widescreen bu-is-6-fullhd">
+                <div class="bu-columns">
+                    <div class="bu-column bu-is-1">
+                        <el-button type="info" plain>Cancel</el-button>
+                    </div>
+                    <div class="bu-column bu-is-1">
+                        <el-button type="success" @click="onApplyFilter">Apply filter</el-button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </cly-section>
+</script>
+
+<script type="text/x-template" id="slipping-away-users-bar-chart">
+    <cly-section>
+        <cly-chart class="chart" :option="slippingAwayUsersOptions" style="height: 400px;" :autoresize="true" v-loading="isLoading"></cly-chart>
+    </cly-section>
+</script>
+
+<script type="text/x-template" id="slipping-away-users-table">
+    <cly-section>
+        <cly-datatable-n  :rows="slippingAwayUsers" :resizable="true" v-loading="isLoading">
+            <template v-slot="scope">
+                <el-table-column sortable="custom" prop="period" :label="i18n('slipping-away-users.table-period')">
+                </el-table-column>
+                <el-table-column sortable="custom" prop="count" :label="i18n('slipping-away-users.table-count')">
+                </el-table-column>
+                <el-table-column sortable="custom" prop="percentage" :label="i18n('slipping-away-users.table-percentage')">
+                    <template slot-scope="scope">
+                        <div class="bu-columns bu-is-gapless">
+                            <div class="bu-column bu-is-2">
+                                <span>{{scope.row.percentage}}%</span>
+                            </div>
+                            <div class="bu-column bu-is-10">
+                                <el-progress :percentage="parseInt(scope.row.percentage)" class="slipping-away-users-td-progress-bar" :show-text="false" :color="progressbarColor"></el-progress>
```

Comment:
As far as I know, we do not use el-progress, but another progress bar provided by Bulma. It is not a Vue component for the time being, but we can wrap it and use it as primary progress component. @itsiprikshit

Reply:
If that's the case then probably it's better we create a component for it. I can always do that.

Reply:
Can you please add this to our Vue wishlist then?